src/Simulation/Simulators.Tests/ResourcesEstimatorTests.cs

```diff
@@ -120,24 +120,40 @@ public void DepthDifferentQubitsTest()
         }
 
         /// <summary>
-        /// Documents that the width and depth statistics reflect independent lower
+        /// Documents that the ExtraWidth and Depth statistics reflect independent lower
         /// bounds for each (two T gates cannot be combined into a circuit of depth
-        /// one and width one).
+        /// one and width one). Width on the other hand is compatible with Depth.
         /// </summary>
         [Fact]
         public void DepthVersusWidthTest()
```

Comment:
The current tests for width/depth (in `Simulators.Tests\Circuits\ResourcesEstimator.qs`) are rather simplistic. Would it make sense to add more coverage? For example, to cover operations that invoke other operations, etc.

Reply:
I already modified DepthVersusWidthTest. I can add another test.